api/scenario_api_utils/scripts/ros_time_publisher.py

```diff
@@ -1,44 +1,49 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
-
-import rospy
+import rclpy
+from rclpy.node import Node
 import time
 from rosgraph_msgs.msg import Clock
 
-
-class RosTimePublisher():
-
+class RosTimePublisher(Node):
     def __init__(self):
-        self.time_rate_ = rospy.get_param("~time_rate", 1.0)
-        # rospy does not have WallTime)
-        self.initial_time_ = rospy.Time.from_sec(time.time())
+        super().__init__('ros_time_publisher')
+        use_sim_time_param = rclpy.parameter.Parameter(
+            "use_sim_time",
+            rclpy.Parameter.Type.BOOL,
+            False)
+        self.set_parameters([use_sim_time_param])
+        self.declare_parameter("time_rate", 1.0)
```

Comment:
Following https://index.ros.org/doc/ros2/Tutorials/Using-Parameters-In-A-Class-Python/, it would be better to set the type explicitly

```py
        my_parameter_descriptor = rclpy.node.Node.ParameterDescriptor(type=ParameterType.PARAMETER_STRING,
                                                                      description='This parameter is mine!')
        self.declare_parameter("my_parameter",
                               "default value for my_parameter",
                               my_parameter_descriptor)
```

Reply:
Created parameter descriptor for time_rate parameter.